llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -10359,6 +10357,8 @@ LoopVectorizeResult LoopVectorizePass::runImpl(
     }
   }
 
+  assert(!Changed || !verifyFunction(F, &dbgs()));
```

Comment:
`Changed` may also be set if the loop only has been modified by `simplifyLoop` or `formLCSSARecursively`, so we will verify in cases where we didn't before?

Reply:
Ah sorry @fhahn, I hadn't appreciated that we might end up verifying even when not vectorising. That's potentially a compile-time regression in some cases then.

Reply:
Sorry about this; fixed now.